src/containers/Certificates/Certificate.tsx

```
@@ -0,0 +1,189 @@
+    const payload = {
+      label: labelvalue,
+      description: descriptionValue,
+      url: urlValue,
+    };
+
+    return payload;
+  };
+
+  const customErrorHandler = (errors: any) => {
+    const { message } = errors[0];
+    if (message && message.includes('Insufficient permissions')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)